meinberlin/apps/votes/templates/meinberlin_votes/token_generation_dashboard.html

```diff
@@ -4,7 +4,7 @@
 {% block dashboard_project_content %}
 <h1 class="u-first-heading">{% translate 'Generate voting codes' %}</h1>
 
-<p class="form-hint u-bottom-divider">{% blocktranslate %}For voting in the third phase, all participants need their own voting code. Here you can generate up to {{ tokens_per_module }} codes. Depending on the number this may take a few minutes.{% endblocktranslate %}</p>
+<p class="form-hint u-bottom-divider">{% blocktranslate %}For voting in the third phase, all participants need their own voting code. Here you can generate up to {{ tokens_per_module }} codes. Depending on the number this may take up to a day.{% endblocktranslate %}</p>
```

Comment:
We could also say "this may take quite a few minutes". :rofl:

Reply:
forgot to change that back to minutes with sha3 :D